apps/ex_wire/lib/ex_wire/util/xor_distance.ex

```diff
@@ -0,0 +1,65 @@
+defmodule ExWire.Util.XorDistance do
```

Comment:
Would it make sense to push this module under `Kademlia`? I think we'll only be using them when using `Kademlia` related stuff and I like to keep utility modules as clean as possible.